### PR TITLE
feat: support exclude paths

### PR DESCRIPTION
**Pull request by EladBezalel**
added options param

related to #2

### CODE REVIEW

**Comment by EladBezalel** [comment]
@erikengervall ping 🙏

**Comment by erikengervall** [comment]
> @erikengervall ping 🙏

Sorry for being super slow, I'm having a look today and got a few simplifications I'd like to get into the PR as well -- it'll likely cause a major bump as it changes the interface

**Comment by erikengervall** [comment]
@EladBezalel thanks for your contribution -- I added a few changes, primarily surrounding unit tests and simplifying the interface of the exported method.

LMK if they make sense and we can move forward!

**Review by erikengervall — APPROVED** [review]
LGTM 🚀

**Comment by EladBezalel** [comment]
Hey! Thanks for the reply and fixes!

As long as you're fine with the breaking changes it LGTM 🙏🏼👍🏼

**Comment by erikengervall** [comment]
> Hey! Thanks for the reply and fixes!
> 
> As long as you're fine with the breaking changes it LGTM 🙏🏼👍🏼

FYI v1.0.0 has been published 🎉

File: src-node/index.ts [hunk 1]
```diff
@@ -15,18 +15,61 @@ export interface FastFindInFiles {
...
+        throw new TypeError('Invalid input: options.excludeFolderPaths.excludeFolderPath must be nonempty string')
+      }
+
+      if (excludeFolderPath.endsWith('/')) {
+        throw new TypeError('Invalid input: options.excludeFolderPaths.excludeFolderPath must not end with "/"')
+      }
+
+      if (!excludeFolderPath.startsWith('./')) {
+        throw new TypeError('Invalid input: options.excludeFolderPaths.excludeFolderPath must start with "./"')
+      }
```
Comment (EladBezalel) [inline src-node/index.ts:66]: Hey @erikengervall, i think this code is wrong,
when i run fast-find-in-files not in the current cwd, this line prevents it (as it doesn't start with `./`.
when i change the code locally to ignore this check, it works,

why did we add this? what does that solve?

Comment (EladBezalel) [reply src-node/index.ts:66]: also, what if the path starts with `../`?

Comment (erikengervall) [reply src-node/index.ts:66]: That's fair, I think I mixed  it up a bit with the trailing `/` restriction -- which did cause issues as it doesn't match with any folder path

We can remove this restriction entirely!

Comment (erikengervall) [reply src-node/index.ts:66]: https://github.com/erikengervall/fast-find-in-files/releases/tag/v1.0.1  removes the unnessary restriciton -- thanks for pointing it out!

Comment (EladBezalel) [reply src-node/index.ts:66]: you rock 👑

**Pull request merged**